packages/contact-manager-rest-api/src/types.ts

```diff
@@ -0,0 +1,24 @@
+import { GenericAuthArgs, ISingleEndpointOpts } from '@sphereon/ssi-express-support'
+import { IContactManager } from '@sphereon/ssi-sdk.contact-manager'
+import { IAgentContext, IDataStore, IKeyManager } from '@veramo/core'
+
+export type ContactManagerMRestApiFeatures =
+  'party_read' |
+  'party_write' |
+  'party_type_read' |
+  'identity_read'
```

Comment:
This has less options than what is being set in the RestAPI on the features array (which will never be set, given current logic, but still). Is this type missing features, or is the RestAPI trying to set features that do no exist (yet)

Reply:
fixed the logic and changed the `contact-manager-api-server`